portfolio/src/main/webapp/scripts/github/index.js

```diff
@@ -0,0 +1,48 @@
+import { github } from "./client.js";
+
+const updateRepos = async (page, per_page) => {
```

Comment:
See https://google.github.io/styleguide/jsguide.html#naming-rules-by-identifier-type - I think in this file a bunch of names you have with _ format should be lowerCamelCase, though let me know if I'm misinterpreting.

Reply:
The only variable I can really rewrite in lowerCamelCase is the per_page. The other variables are written just as Github API send me the info.

I rewriter all of them to lowerCamelCase and corrected the function parameter `per_page`

Reply:
Acknowledged, sounds good!